infra/main.tf

```
@@ -0,0 +1,694 @@
+terraform {
+  required_providers {
+    google = {
+      source = "hashicorp/google"
+      version = "3.48.0"
+    }
+    kubernetes = {
+      source = "hashicorp/kubernetes"
+      version = "1.13.3"
+    }
+  }
+}
+
+variable "gsuite_organization" {}
+variable "batch_gcp_regions" {}
+variable "gcp_project" {}
+variable "batch_logs_bucket_location" {}
+variable "batch_logs_bucket_storage_class" {}
+variable "gcp_region" {}
+variable "gcp_zone" {}
+variable "domain" {}
+
+provider "google" {
+  credentials = file("~/.hail/terraform_sa_key.json")
+
+  project = var.gcp_project
+  region = var.gcp_region
+  zone = var.gcp_zone
+}
+
+data "google_client_config" "provider" {}
+
+resource "google_project_service" "service_networking" {
+  service = "servicenetworking.googleapis.com"
+}
+
+resource "google_compute_network" "default" {
+  name = "default"
+}
+
+data "google_compute_subnetwork" "default_region" {
+  name = "default"
+  region = var.gcp_region
+  depends_on = [google_compute_network.default]
+}
+
+resource "google_container_cluster" "vdc" {
+  name = "vdc"
+  location = var.gcp_zone
+  network = google_compute_network.default.name
+
+  # We can't create a cluster with no node pool defined, but we want to only use
+  # separately managed node pools. So we create the smallest possible default
+  # node pool and immediately delete it.
+  remove_default_node_pool = true
+  initial_node_count = 1
+
+  master_auth {
+    username = ""
+    password = ""
+
+    client_certificate_config {
+      issue_client_certificate = false
+    }
+  }
+}
```

What do you think about using a release channel for the GKE cluster?

```suggestion
  release_channel {
    channel = "STABLE"
  }
}
```